src/migrations/kotlin/io/easybreezy/migrations/V20200512092106__NeutralPriorityColor.kt

```diff
@@ -12,4 +12,8 @@ class V20200512092106__NeutralPriorityColor : BaseJavaMigration() {
             """.trimIndent()
         )
     }
+
+    override fun canExecuteInTransaction(): Boolean {
```

Comment:
What does it do?

Reply:
In postgres you can not do ALTER TYPE in transaction if this type already used in some table.
I've had similar migration and this problem.
For dev i think it's ok. On prod migration will be executed on empty database.

Reply:
But how we can divide prod and dev migrations?

Reply:
I meant that there is no problem, because we don't have prod with some data. It does not matter that migration will be executed without transaction now or later.

Reply:
In this case, will be better to remove all data in a transaction before executing migration query

Reply:
> if this type already used in some table
table without data also throws postgres exception